### DISCUSSION TITLE
How to replicate force404 when using cy.intercept

### DISCUSSION THREAD

**Discussion by John-Ender** [cypress-io/cypress#15436] [Component Testing]
We want to force all non-mocked API calls to return 404 for our tests. With the `cy.server` and `cy.route` this was simple. After upgrading to Cypress 6.0 and migrating all of our `cy.route` calls to  `cy.intercept` there doesn't seem to be any documentation on how to achieve the same effect. 

How can I replicate the global 404 response for non mocked calls?

**Comment by bahmutov** [comment]
Would something like this https://glebbahmutov.com/blog/cypress-intercept-problems/#force404 work?
